### ISSUE TITLE
Protractor waits for pending 101 requests

### ISSUE THREAD

**Issue by diegohq** [angular/protractor#4147]
For some reason in Chrome console network tab there is a websocket request as "Pending" so apparently Pratractor keeps waiting this request to finish. This request has 101 HTTP status after a quick search I learned that it is suposed to be marked as "Pending" in Chrome console.

I believe Protractor should not wait for this request to finish once it will never finish. A workaround is to prevent syncronization:

`browser.ignoreSynchronization = true;`

And defined a timeout in configuration file:

`allScriptsTimeout: 50000`

I am using Protractor 5.1.1, Node 6.9.4, NPM 4.0.5; the application uses Angular 2.

**Comment by NickTomlin** [comment]
Hi there!

Your question is better suited for StackOverflow or Gitter. Please [ask a question there with the 'protractor' tag](http://stackoverflow.com/questions/ask?tags=protractor) or post in the [Gitter Channel](https://gitter.im/angular/protractor) to get help.

From the the [getting help](https://github.com/angular/protractor#getting-help) section of the README:

> Please ask usage and debugging questions on StackOverflow (use the "protractor" tag) or in the Angular discussion group. (Please do not ask support questions here on Github.)

Thanks!

**Comment by diegohq** [comment]
Actually is not a question, I believe it is a bug.

But if it is the expected behavior that's ok! ;)

**Comment by NickTomlin** [comment]
Hi there, 

from your issue it's unclear that this is a bug with protractor itself. It sounds like the site your are testing is making a websocket request (or are you saying that this is happening by default because you are opening the chrome network tab?) and causing protractor to hang. Because of this, it may be an issue isolated to your setup and not something it is necessarily wrong withProtractor. 

If you can produce a small reproducible example and believe this is an issue with protractor itself i'd be happy to re-open the issue. Otherwise, it sounds like it's something that you'll need to resolve on your own. 

I hope that makes sense, thanks!

**Comment by diegohq** [comment]
Hello,

Sure, no problem I understand... I believe it is an error because the request responds with 101 HTTP Code and as far as I researched it is expected that Chrome console indicates it as "processing" forever even though the request is completed.

However it can be a problem in my application. Actually I am new in Angular and using a template so I don't know exactly why and where this request is made. Since I got this "processing forever" information on Google I thought maybe Protractor could threat. But, again, I'm new at this and may be a problem in my application. ;)

Thanks!
